src/postamble.js

```diff
@@ -125,6 +125,10 @@ Module['then'] = function(func) {
 };
 #endif
 
+/**
```

Comment:
do these comments work? the js optimizer runs before closure, and it would strip them, I believe.